corehq/apps/userreports/views.py

```diff
@@ -167,6 +168,65 @@ def section_url(self):
         return reverse(ReportBuilderTypeSelect.urlname, args=[self.domain])
 
 
+class ReportBuilderPaywallBase(BaseDomainView):
+    page_title = ugettext_lazy('Subscribe')
+
+    @property
+    def section_name(self):
+        return _("Report Builder")
```

Comment:
`ugettext_lazy()` is used at the class level because if we translate it there with `ugettext()`, it will only be translated at the time that this module is imported, not for each request. And then I used `ugettext` in the other place because that's my sort of "default" translation method.